CONTRIBUTING.md

```diff
@@ -18,12 +18,25 @@ Our Rust projects use the standard Rust coding style, and should all have tests.
 First, make sure you have an up-to-date stable Rust, with the necessary tools. You can install Rust using `rustup`:
 
 ```sh
+
+# If you have linux system rust installed, remove it first
+sudo apt-get remove rustc
```

Comment:
I _think_ that if you install `rustup` correctly, it should be at the front of your `PATH` and override your system Rust.